### PR TITLE
 Display a message in pledges status page

### LINKED ISSUE #282: Need to display a message when having an error in fetching pledges or there are no pledges in an aid package

### PR DESCRIPTION
**Pull request by ManojLL**
## Purpose

The purpose of this PR is to fix #282 

## Goals

Display a message on the pledges status page when having an error in fetching pledges or there are no pledges in an aid package
### Screenshots

Screenshot from 2022-07-28 01-05-22

### CODE REVIEW

File: admin-portal/src/pages/pledgeStatus/pledgeStatus.tsx [hunk 1]
```diff
@@ -56,36 +56,45 @@ export default function PledgeStatus() {
...
-            </div>
-          </div>
-          <DonorTable
-            pledges={pledges}
-            onPledgeEdit={handlePledgeEdit}
-            onPledgeDelete={handlePledgeDelete}
-          />
+          ) : (
+            <p className="errorMessage">
+              This AidPackage doesn&apos;t contain Pledge yet
```
Comment (jayasanka-sack) [inline admin-portal/src/pages/pledgeStatus/pledgeStatus.tsx:95]: Render this inside the `DonorTable` component. 

Display a single row with an error message if there are no items. 

How about using the following text?

"This AidPackage does not yet include any pledges."

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]
Thanks @ManojLL! I added a small comment

File: admin-portal/src/pages/pledgeStatus/pledgeStatus.css [hunk 2]
```diff
@@ -10,3 +10,10 @@
   display: flex;
   align-items: center;
 }
+
+.errorMessage{
+  margin-top: 10rem;
+  text-align: center;
+  justify-content: center;
+  font-size: 3vh
+}
```
Comment (anjula-sack) [inline admin-portal/src/pages/pledgeStatus/pledgeStatus.css:19]: Can you add the end of the line?

**Review by anjula-sack — CHANGES_REQUESTED** [review]

**Review by anjula-sack — CHANGES_REQUESTED** [review]
Can you fix the lint error?

File: admin-portal/src/pages/pledgeStatus/pledgeStatus.css [hunk 3]
```diff
@@ -9,4 +9,4 @@
 .pledgeStatus .contributionsSummary {
   display: flex;
   align-items: center;
-}
+}
```
Comment (anjula-sack) [inline admin-portal/src/pages/pledgeStatus/pledgeStatus.css:12]: Add a new line here

**Review by anjula-sack — APPROVED** [review]
Thank you! @ManojLL

**Pull request merged**